Sage_framework/build_sage.sh

```diff
@@ -53,66 +65,127 @@ else
 	export LDFLAGS="-Wl,-platform_version,macos,10.12,11.3"
     fi
     export MACOSX_DEPLOYMENT_TARGET="10.12"
+    export CONDA_OVERRIDE_OSX=10.13
 fi
+
+OPTIONAL_PKGS=" \
+isl \
+4ti2 \
+benzene \
+bliss \
+gap_packages \
+latte_int \
+bliss \
+buckygen \
+cbc \
+coxeter3 \
+sagemath_coxeter3 \
+csdp \
+e_antic \
+frobby \
+igraph \
+kenzo \
+libnauty \
+libsemigroups \
+lrslib \
+meataxe \
+sagemath_meataxe \
+mcqd \
+mpfrcx \
+normaliz \
+p_group_cohomology \
+pari_elldata \
+pari_galpol \
+pari_nftables \
+plantri \
+sagemath-bliss \
+sage_numerical_backends_coin \
+pynormaliz \
+pycosat \
+pysingular \
+qepcad \
+sirocco \
+sagemath_sirocco \
+symengine \
+symengine_py \
+tdlib \
+tides "
+
+# Disable some conda packages. This is strictly not necessary
+# as they get overriden by the SPKGS, but good to keep track
+
+# These packages don't have osx-arm64 packages
+DISABLE_CONDA="4ti2 latte_int lrslib pynormaliz pysingular"
+# These depend on sagelib
+DISABLE_CONDA="$DISABLE_CONDA sagemath_sirocco"
+# No spkg-configure.m4
+DISABLE_CONDA="$DISABLE_CONDA e_antic elliptic_curves"
+# lrsnash missing
+DISABLE_CONDA="$DISABLE_CONDA lrslib"
+# spkg-configure.m4 insufficient
+DISABLE_CONDA="$DISABLE_CONDA mathjax threejs"
+# not sure what's going on here
+DISABLE_CONDA="$DISABLE_CONDA pythran symengine"
+
+CONDA_PKGS="autoconf automake libtool pkg-config"
```

Comment:
@culler to use python=3.13
```suggestion
CONDA_PKGS="autoconf automake libtool pkg-config python=3.13"
```